stackcopy.py

```
@@ -230,98 +249,168 @@ def main():
     try:
         for entry in os.scandir(src_dir):
             if entry.is_file():
-                # Split the filename into stem and extension
                 stem, ext = os.path.splitext(entry.name)
-                # Check if the extension is a raw format (case-insensitive)
-                if ext.lower() in RAW_EXTENSIONS:
-                    # Add the stem to the set (lowercase for case-insensitive matching)
-                    raw_stems.add(stem.lower())
+                ext_lower = ext.lower()
+
+                if not stem in file_db:
```

Ruff (0.14.2)

Test for membership should be `not in`

Convert to `not in`

(E713)